src/pydrex/io.py

```diff
@@ -0,0 +1,141 @@
+"""PyDRex: Input/Output functions and helpers."""
+import configparser
+import pathlib
+import runpy
+
+import numpy as np
+
+import pydrex.exceptions as _err
+
+
+def parse_params(file):
+    """Parse an INI file containing PyDRex parameters."""
+    config = configparser.ConfigParser()
+    configpath = _resolve_path(file)
+    config.read(configpath)
+
+    geometry = config["Geometry"]
+    output = config["Output"]
+    params = config["Parameters"]
+
+    mesh = read_mesh(_resolve_path(geometry.get("meshfile"), configpath.parent))
+    olivine_fraction = params.getfloat("olivine_fraction")
+    enstatite_fraction = params.getfloat("enstatite_fraction", 1.0 - olivine_fraction)
+
+    # TODO: Allow user-given order for Euler angle output (like Fraters 2021)?
+    olivine_output = tuple(
+        output.get("olivine", fallback="volume_fraction, rotation_matrix")
+        .replace(" ", "")
+        .split(",")
+    )
+    enstatite_output = tuple(
+        output.get("enstatite", fallback="volume_fraction, rotation_matrix")
+        .replace(" ", "")
+        .split(",")
+    )
+
+    return dict(
+        zip(
+            [
+                "mesh",
+                "simulation_name",
+                "checkpoint_interval",
+                "olivine_output",
+                "enstatite_output",
+                "olivine_fraction",
+                "enstatite_fraction",
+                "stress_exponent",
+                "gbm_mobility",
+                "gbs_threshold",
+                "nucleation_efficiency",
+                "olivine_fabric",
+                "number_of_grains",
+            ],
+            [
+                mesh,
+                output.get("simulation_name", fallback="simulation_name"),
+                output.getint(
+                    "checkpoint_interval", fallback=0
+                ),  # 0 disables checkpointing
+                ["olivine_" + s for s in olivine_output],  # Unique names for np.savez()
+                [
+                    "enstatite_" + s for s in enstatite_output
+                ],  # Unique names for np.savez()
```

Comment:
This list should be formatted similarly to the previous one.

Reply:
Ah, that's due to running [black](https://pypi.org/project/black/). I'll change it, don't need to repeat the comment anyway.